{
  "_args": [
    [
      {
        "raw": "react-transform-catch-errors@^1.0.2",
        "scope": null,
        "escapedName": "react-transform-catch-errors",
        "name": "react-transform-catch-errors",
        "rawSpec": "^1.0.2",
        "spec": ">=1.0.2 <2.0.0",
        "type": "range"
      },
      "/workspaces/react-node-socket-ejs-demo"
    ]
  ],
  "_from": "react-transform-catch-errors@>=1.0.2 <2.0.0",
  "_id": "react-transform-catch-errors@1.0.2",
  "_inCache": true,
  "_location": "/react-transform-catch-errors",
  "_nodeVersion": "5.1.1",
  "_npmOperationalInternal": {
    "host": "packages-5-east.internal.npmjs.com",
    "tmp": "tmp/react-transform-catch-errors-1.0.2.tgz_1454433770011_0.6742109549231827"
  },
  "_npmUser": {
    "name": "gaearon",
    "email": "dan.abramov@gmail.com"
  },
  "_npmVersion": "3.7.0",
  "_phantomChildren": {},
  "_requested": {
    "raw": "react-transform-catch-errors@^1.0.2",
    "scope": null,
    "escapedName": "react-transform-catch-errors",
    "name": "react-transform-catch-errors",
    "rawSpec": "^1.0.2",
    "spec": ">=1.0.2 <2.0.0",
    "type": "range"
  },
  "_requiredBy": [
    "/",
    "/babel-preset-react-hmre"
  ],
  "_resolved": "https://registry.npmjs.org/react-transform-catch-errors/-/react-transform-catch-errors-1.0.2.tgz",
  "_shasum": "1b4d4a76e97271896fc16fe3086c793ec88a9eeb",
  "_shrinkwrap": null,
  "_spec": "react-transform-catch-errors@^1.0.2",
  "_where": "/workspaces/react-node-socket-ejs-demo",
  "author": {
    "name": "Dan Abramov",
    "email": "dan.abramov@me.com"
  },
  "bugs": {
    "url": "https://github.com/gaearon/react-transform-catch-errors/issues"
  },
  "dependencies": {},
  "description": "React Transform that catches errors inside React components",
  "devDependencies": {
    "babel-cli": "^6.3.17",
    "babel-core": "^6.3.21",
    "babel-preset-es2015": "^6.3.3",
    "babel-preset-stage-0": "^6.3.13",
    "rimraf": "^2.4.3"
  },
  "dist": {
    "shasum": "1b4d4a76e97271896fc16fe3086c793ec88a9eeb",
    "tarball": "https://registry.npmjs.org/react-transform-catch-errors/-/react-transform-catch-errors-1.0.2.tgz",
    "integrity": "sha512-FCeTXLTpRM8wMrwAlYVoEI+HXUFYEJLKdmf9zd5YtH1yc2Dq81JL1YMr0eHeUe699CCTezG4CfEB5D9RpmovEw==",
    "signatures": [
      {
        "keyid": "SHA256:jl3bwswu80PjjokCgh0o2w5c2U4LhQAE57gj9cz1kzA",
        "sig": "MEQCIGXtKP4QtFPtYhR6ijx8yOmcFPQ8Xp8QmMrNcNbboZaEAiBTfSgN5Szuv1Dg9ohjmAiJRmFo9BgjaU7HzegMtYsvVA=="
      }
    ]
  },
  "files": [
    "lib",
    "src"
  ],
  "gitHead": "ffa576dd1d38399613aa7d8293d9bc0bdfd43594",
  "homepage": "https://github.com/gaearon/react-transform-catch-errors#readme",
  "keywords": [
    "react-transform",
    "react",
    "reactjs",
    "errors",
    "rhl",
    "dx"
  ],
  "license": "MIT",
  "main": "lib/index.js",
  "maintainers": [
    {
      "name": "gaearon",
      "email": "dan.abramov@gmail.com"
    },
    {
      "name": "thejameskyle",
      "email": "me@thejameskyle.com"
    }
  ],
  "name": "react-transform-catch-errors",
  "optionalDependencies": {},
  "readme": "ERROR: No README data found!",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/gaearon/react-transform-catch-errors.git"
  },
  "scripts": {
    "build": "babel src --out-dir lib",
    "clean": "rimraf lib",
    "prepublish": "npm run clean && npm run build"
  },
  "version": "1.0.2"
}
